app/packages/core/src/components/ColorModal/FieldSetting.tsx

```diff
@@ -40,7 +40,8 @@ type State = {
 };
 
 const FieldSetting: React.FC<Prop> = ({ prop }) => {
-  const colorContainer: React.RefObject<HTMLDivElement> = React.createRef();
+  const wrapperRef: React.RefObject<HTMLDivElement> = React.createRef();
+  const pickerRef: React.RefObject<TwitterPicker> = React.createRef();
```

Comment:
Why not `useRef()`? Is it intentional that you want these refs to not be memoized between renders?

Reply:
changing to useRef()